services/minespace-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -672,156 +692,166 @@ const renderUploadInitialNotificationDocuments = (
 
   return (
     <Row>
-      {!formDisabled && (
-        <>
-          <Col span={24}>
-            <Typography.Title level={4}>Upload Initial Notification Documents</Typography.Title>
-            <Typography.Paragraph>
-              Please upload any initial notifications that will provide context with this incident
-              report.
-            </Typography.Paragraph>
-          </Col>
-          <Col span={24}>
-            <Form.Item>
-              <Field
-                id={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
-                name={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
-                onFileLoad={(document_name, document_manager_guid) =>
-                  handlers.onFileLoad(
-                    document_name,
-                    document_manager_guid,
-                    Strings.INCIDENT_DOCUMENT_TYPES.initial,
-                    INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD
-                  )
-                }
-                onRemoveFile={parentHandlers?.deleteDocument}
-                mineGuid={props.match.params?.mineGuid}
-                component={IncidentFileUpload}
-                labelIdle='<strong class="filepond--label-action">Supporting Document Upload</strong><div>Accepted filetypes: .kmz .doc .docx .xlsx .pdf</div>'
-              />
-            </Form.Item>
-          </Col>
-        </>
-      )}
-      {props.formValues?.documents?.length > 0 && (
-        <Col span={24}>
-          {formDisabled && (
-            <Typography.Title level={3} id="documentation">
-              {title}
-            </Typography.Title>
-          )}
-          <Row>
-            <Col xs={24} md={12}>
-              <Typography.Title level={4}>{subTitle}</Typography.Title>
+      {(!props.formValues?.documents || props.formValues?.documents?.length === 0) &&
```

Comment:
Could we consolidate these conditional checks into named variables? I hate to be that guy, but trying to troubleshoot this later could be a bit tricky with how complicated this is. Give me a shout if you want to talk through it.